lib/gorgon/command.rb

```diff
@@ -0,0 +1,137 @@
+require "rubygems"
+require 'gorgon'
+require 'gorgon/originator'
+require 'gorgon/listener'
+require 'gorgon/rsync_daemon'
+require 'gorgon/worker_manager'
+require 'gorgon/ping_service'
+require 'gorgon/gem_service'
+require 'gorgon/version'
+require 'gorgon/listener_installer'
+require 'gorgon/settings/initial_files_creator'
+
+module Gorgon
+  class Command
+    attr_reader :argv
```

Comment:
I would put `attr_reader` after the constant definitions to be consistent with the [community style guide](https://github.com/bbatsov/ruby-style-guide#consistent-classes)